docs/docs/get-started/quickstart.md

```diff
@@ -43,6 +44,10 @@ mamba install nebari
   </TabItem>
 </Tabs>
 
+:::note
+The Nebari CLI will auto-detect and then deploy any [Nebari Extensions][nebari-extension-system] that are installed in your Python environment, and extensions once deployed cannot be uninstalled our automatically destroyed with a `nebari destroy` command.  For this reason, we recommend creating a [unique environment][environment-management] for each Nebari deployment, especially when managing multiple deployments with extensions.
```

Comment:
I think this is a good place for this, I would just beware of the destroy part as stated above

Reply:
Removed destroy reference